specviz/core/plugin.py

```diff
@@ -56,13 +56,6 @@ def get_action(parent, level=None):
 
 
 class Plugin(DecoratorRegistry):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        self._plugin_bar_decorator = PluginBarDecorator()
-        self._tool_bar_decorator = ToolBarDecorator()
-        self._plot_bar_decorator = PlotBarDecorator()
```

Comment:
Was all of this just extraneous?

Reply:
Yep. I'm quite sure that I actually made the change to remove this stuff in an earlier PR that I closed for some reason, and forgot to make the changes to a later PR.